lemminx-maven/src/test/java/org/eclipse/lemminx/maven/test/PluginResolutionTest.java

```diff
@@ -67,9 +70,16 @@ public void testPluginConfigurationHoverMissingTransitiveDependency()
 			throws IOException, InterruptedException, ExecutionException, URISyntaxException {
 		assertFalse(initialMavenPluginApiDirectory.exists());
 		// <compilerArguments> hover
+		
+		SharedSettings settings = new SharedSettings();
```

Comment:
@mickaelistria do you think a JUnit rule or something should be used here? To avoid duplicating the markdown hover settings

Reply:
Maybe, but I don't think it's worth bothering too much about it.